client/packages/common/src/intl/locales/tet/replenishment.json

```
@@ -75,7 +75,8 @@
   "messages.select-rows-to-delete-them": "Hili lina sira atu hamos",
   "messages.shipment-saved": "Remessa salva ona🥳",
   "label.min": "Minimu",
-  "messages.confirm-inbound-status-as": "Konfirma estatus nudar $t({{estatus}})?\nNotas: Muda husi 'foun' sei hamos lina ne'ebe ho kuantidade Zero.",
+  "messages.confirm-changing-from-new": "\nNotas: Muda husi 'foun' sei hamos lina ne'ebe ho kuantidade Zero.",
```

The newline char isn't required now
```suggestion
  "messages.confirm-changing-from-new": "Notas: Muda husi 'foun' sei hamos lina ne'ebe ho kuantidade Zero.",
```